cirq/google/serializable_gate_set.py

```diff
@@ -13,14 +13,21 @@
 # limitations under the License.
 """Support for serializing and deserializing cirq.api.google.v2 protos."""
 
-from typing import cast, Dict, Iterable, Optional, Tuple, Union
+from collections import defaultdict
+
+from typing import (cast, Dict, Iterable, Optional, Tuple, Type, TYPE_CHECKING,
+                    Union)
 
 from google.protobuf import json_format
 
 from cirq import circuits, devices, ops, schedules, value
 from cirq.api.google import v2
 from cirq.google import op_deserializer, op_serializer
 
+if TYPE_CHECKING:
```

Comment:
do we still have to do this in python 3?

Reply:
nope, fixed.